Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,44 @@
+public class View: Viewable {
+	var ui: Receivable 
+	var guess: String!
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
+		guess = ui.getUserInput()
+		if (containsOnlyLetters(guess)) {
+			return guess
+		} else {
+			print("Invalid guess. Please enter a lowercase letter or word.")
+			receiveGuess()
+		}
+		return guess
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) -> String {
```

Comment:
the logic here is like, basically the same as in the mock, the mock/stub object should be void of logic.